Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -205,26 +237,47 @@ void __stdcall PlayerLaunch(unsigned int iShip, unsigned int client)
 
 	if (mobiledockClients[client].mobileDocked)
 	{
-
 		returncode = SKIPPLUGINS;
 
-		// Update the location of the carrier and remove the docked ship from the carrier
-		if (carrier_client != -1)
+		// Set last base to last real base this ship was on. If this was POB... well, base plugin should export more functions to work with.

```

Comment:
I think it does now - see the latest commit to the repo, regarding GetCustomLastBaseForClient/CUSTOM_BASE_LAST_DOCKED

Reply:
Let's make this into a TODO to implement support for at a later stage.